packages/renderers/renderer-preact/index.js

```diff
@@ -2,5 +2,14 @@ export default {
   name: '@astrojs/renderer-preact',
   client: './client',
   server: './server',
-  knownEntrypoints: ['preact', 'preact-render-to-string'],
+  knownEntrypoints: ['preact', 'preact/jsx-runtime', 'preact-render-to-string'],
+  jsxImportSource: 'preact',
+  jsxTransformOptions: async () => {
+    const { default: { default: jsx }} = await import('@babel/plugin-transform-react-jsx');
+    return {
+      plugins: [
+        jsx({}, { runtime: 'automatic', importSource: 'preact' })
```

Comment:
This is the new JSX transform that uses `preact/jsx-runtime`!

Reply:
nice!